### DISCUSSION TITLE
Adding to a home/ acting as a speaker

### DISCUSSION THREAD

**Discussion by dallman2** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#485] [Q&A]
Hey all,

I would like to add the pc this is running on to the home I have created with some smart lights/ tvs/ etc. How do I do this? GAUDC is able to toggle my lights on and off when I ask it to but I cannot see it in my Home App...

**Comment by im-coder-lg** [comment]
It is because of the official Google Assistant SDK it uses. Google did that.

**Reply by MadmaxHart** [reply]
So this will never show up in the Google Home App? Is it possible to set this to a room (e.g. Computer Room)?

Thanks
